ballerina/build.gradle

```diff
@@ -96,6 +96,7 @@ publishing {
 
 updateTomlFiles.dependsOn copyStdlibs
 
+build.doNotTrackState("build needs to re-run every time")
```

Comment:
Why are we re-running the build every time? One of the main reasons to use Gradle was the build caching, which improves the build time and makes the life easy for the developers.